src/components/editTestCase/qdm/EditTestCase.tsx

```diff
@@ -191,6 +185,14 @@ const EditTestCase = () => {
 
   const updateTestCase = async (testCase: TestCase) => {
     const modifiedTestCase = { ...currentTestCase, ...testCase };
+    const errorMsg = checkSpecialCharacters(modifiedTestCase);
+    if (errorMsg) {
+      showToast(
+        `Error updating Test Case for measure: "${measure.measureName}": ${errorMsg}`,
```

Comment:
how do I know which character is not allowed?

Reply:
@adongare The error message should say something like "Test Case Title/Group cannot have special characters". Do you suggest presenting a full list of restricted characters?

Reply:
@adongare I added the list to the message

Reply:
yeah I think so. ticket says to conclude special character set in message